opi_pc_plus/opi_pc_plus.mk

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2020 Roman Stratiienko (r.stratiienko@gmail.com)
```

Comment:
Please change all such lines to:
Copyright (C) 2021 GloDroid project

Reply:
Done.